src/main/java/com/nfl/glitr/registry/type/Scalars.java

```diff
@@ -41,6 +41,17 @@ public Object serialize(Object input) {
                 return formatter.format(time);
             }
 
+            if (input instanceof String) {
+                String time = (String) input;
+
+                try {
+                    Instant instant = Instant.parse(time);
+                    return serialize(instant);
+                } catch (RuntimeException ex) {
```

Comment:
I'd remove the catch block if you don't do anything with it. Otherwise you can add some context information to the parse error.